requirements-dev.txt

```diff
@@ -133,7 +135,7 @@ pytest-env==0.6.2
     # via -r requirements-dev.in
 pytest-mock==3.5.1
     # via -r requirements-dev.in
-pytest-split==0.3.3
+pytest-split==0.6.0
```

Comment:
Interesting this was out of sync. Who updated this last?

Reply:
GH seems to suggest it was you? 👀

Reply:
:see_no_evil: :hear_no_evil: :speak_no_evil: